src/app/app.config.ts

```diff
@@ -46,6 +46,28 @@ export const INSPIRE_EDITOR_CONFIG: AppConfig = {
     'external_system_numbers.items.properties.value': {
       x_editor_priority: 1
     },
+    'accelerator_experiments.items.properties.experiment': {
+      x_editor_autocomplete: {
+        url: 'http://localhost:5000/api/experiments/_suggest?experiment=',
```

Comment:
Once we merge https://github.com/inspirehep/record-editor/pull/90

Might be better to use `BASE_URL` (https://github.com/inspirehep/record-editor/pull/90/files#diff-35a6e7360edca9cd9bb3ec02d53d1273R114) so that when we deploy the code outside of localhost it works.